scripts/deploy/deployConfig.sh

```diff
@@ -31,7 +31,7 @@ BYTECODE_STORAGE_PATH="deployments/_bytecode_storage.json"
 # any networks listed here will be excluded from actions that are applied to "all networks"
 # exclude all test networks:       EXCLUDE_NETWORKS="bsctest,goerli,sepolia,mumbai,consensys-zkevm-testnet"
 # exclude all production networks: EXCLUDE_NETWORKS="mainnet,polygon,bsc,gnosis,fantom,okx,avalanche,arbitrum,optimism,moonriver,moonbeam,celo,fuse,cronos,velas,harmony,evmos,aurora,boba,nova"
-EXCLUDE_NETWORKS="bsctest,goerli,sepolia,mumbai,consensys-zkevm-testnet"
+EXCLUDE_NETWORKS="gnosis,okx,moonbeam,celo,fuse,cronos,velas,harmony,evmos,boba,nova,bsctest,goerli,sepolia,mumbai,consensys-zkevm-testnet"
```

Comment:
In general it would be good to change this back to old value before merge but I will just do that with my next PR. Just keep in mind in case you deploy something before I open next PR